Dockerfile

```
@@ -1,71 +1,103 @@
-COPY --from=builder /app/next.config.ts ./
-COPY --from=builder /app/prisma/generated ./prisma/generated
-
-# Create non-root user
-RUN addgroup --system --gid 1001 nodejs
-RUN adduser --system --uid 1001 nextjs
-
-# Change ownership of the app directory
-RUN chown -R nextjs:nodejs /app
+ENV NEXT_TELEMETRY_DISABLED=1
+RUN pnpm version:sync
+RUN DATABASE_URL="${DATABASE_URL}" DATABASE_DIRECT_URL="${DATABASE_DIRECT_URL:-${DATABASE_URL}}" \
```

Hadolint (2.15.1)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)